fuzz_test/fuzz_tester.cpp

```diff
@@ -1,20 +1,18 @@
-#include <fmt/format.h>
-#include <iterator>
-#include <utility>
+#include <stdint.h>
```

Comment:
cstdint, cstddef

Reply:
I don't wanted to change the original test from libFuzzer itself.